docs/notebooks/proton_radiography_particle_tracing.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<p><code>sim.synthetic_radiograph</code>could definitely use xarray outputs... I'll tinker with it a little.</p><p> </p><p>I guess <code>bins</code>  is the number of bins in the horizontal and vertical directions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Yes, that keyword is just directly passed on to np.histogram2d but I can provide more explanation here if that's not a commonly used fcn.